app/controllers/new_administrateur/procedures_controller.rb

```diff
@@ -186,7 +186,12 @@ def transfer
     end
 
     def invited_expert_list
-      @invited_expert_emails = ExpertsProcedure.invited_expert_emails(@procedure)
+      @experts_procedure = ExpertsProcedure.invited_experts(@procedure)
```

Comment:
```suggestion
      @experts = @procedure.experts
```

Il te manquera peut être un `sort` . Du coup j'ai l'impression que la méthode `ExpertsProcedure.invited_experts` ne sert plus.

Reply:
Dans ce cas j'ai besoin instances de ExpertsProcedure (et non pas experts) car je boucle dedans pour rendre notifiable les experts.